conference/react-js/src/components/SettingsPage/style.scss

```diff
@@ -0,0 +1,116 @@
+.settings-wrapper {
+  width: 50%;
+  height: calc(100vh - 90px);
+  margin: 20px auto;
+  box-sizing: border-box;
+  padding: 20px;
+  box-shadow: (0, 0, 5px, 0, rgba(0,0,0,0.15));
+
+  @media ($mnw: $mdmn) and ($mxw: $xmdmx) {
+    width: 90%;
+    }
+  @media ($mnw: $xsdmn) and ($mxw: $sdmx) {
+    width: 100%;
+    padding: 20px 10px;
+    }
+  @media ($mnw: $xmmn) and ($mxw: $ldmx) {
+    width: 80%;
+    }
+  }
+
+.settings {
+  &__success {
+    background: $lime-green
+    }
+
+  &__error {
+    background: $raspberry
+    }
+
+  &__info {
+    padding: 20px;
+    text-align: center;
+    }
+
+  &__row {
+    display: flex;
+    margin: 20px 0;
+    }
+
+  &__header {
+    border-bottom: 1px solid $sharp-blue;
+    text-transform: capitalize;
+    box-sizing: border-box;
+    padding-bottom: 5px;
+    }
+
+  &__title {
+    width: 30%;
+    text-transform: capitalize;
+    @media ($mnw: $xsdmn) and ($mxw: $sdmx) {
+      width: 100%;
+      }
+    }
+
+  &__row-content {
+    font-family: Oswald Light, sans-serif;
+    width: 70%;
+    &_name {
+      overflow: hidden;
+      text-overflow: ellipsis;
+      }
+    }
+
+  &__edit {
+    width: 6%;
+    text-align: right;
+    text-transform: capitalize;
+    }
+
+  &__brief {
+    display: flex;
+    justify-content: flex-start;
+    flex-wrap: wrap;
+    box-sizing: border-box;
+    padding: 10px;
+    border-bottom: 1px solid $light-grey;
+
+    &:hover {
+      cursor: pointer;
+      background: #f2f2f2;
+      }
+
+    &:active {
+      outline: none
+      }
+
+    &:hover .settings__edit {
+      text-decoration: underline;
+      @media ($mnw: $mdmn) {
+        &:before {
+          content: '';
+          display: block;
+          width: 15px;
+          height: 15px;
+          float: left;
+          //background: url(assets/img/pen-icon.svg) no-repeat center;
```

Comment:
clean?

Reply:
I just forgot to include the image to the final version.